index.css

```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
```

Comment:
Missing a css selector?
did you mean:
* 
also css should not be all on one line